frontend/src/pages/projects/screens/detail/ProjectDetails.tsx

```diff
@@ -0,0 +1,48 @@
+import { Divider, PageSection, Stack, StackItem } from '@patternfly/react-core';
+import * as React from 'react';
+import { useParams } from 'react-router';
+import ApplicationsPage from '../../../ApplicationsPage';
+import EmptyProjects from '../../EmptyProjects';
+import DataConnectionsList from './DataConnectionsList';
+import ModelServingList from './ModelServingList';
+import ProjectDetailsSidebarWrapper from './ProjectDetailsSidebar';
+import StorageList from './StorageList';
+import WorkspacesList from './WorkspacesList';
+
+const ProjectDetails: React.FC = () => {
+  const { namespace } = useParams<{ namespace: string }>();
+
+  return (
+    <ApplicationsPage
+      title={`${namespace} project details`}
+      description={null}
+      loaded
+      empty={false}
+      emptyStatePage={<EmptyProjects />}
+    >
+      <PageSection id="project-details-list" hasOverflowScroll variant="light">
+        <ProjectDetailsSidebarWrapper>
+          <Stack hasGutter>
+            <StackItem>
+              <WorkspacesList />
```

Comment:
Almost want to just map over them... boilerplate between each one is kinda error prone.

```
const sections = [
  // component could even be data based on my previous comment about boilerplate code 
  { id: '...enum value...', component: <WorkspacesList /> },
  { id: '...enum value...', component: <StorageList /> },
  ...
];

<Stack hasGutter>
  {sections.map(({ id, component }) => (
    <React.Fragement key={id}>
      <StackItem>
        {component}
      </StackItem>
      <Divider />
    </React.Fragment>
  )}
</Stack>
```